examples/Calibrate/Calibrate.ino

```diff
@@ -0,0 +1,21 @@
+#include <Servo.h>
+Servo myservo;     // create servo object to control a servo
+int input;         // variable to read the value from the SerialMonitor 
+int servoPin = 11; // set the Pin on Arduino, servo is attached
+
+
+void setup() {
+  myservo.attach(servoPin); // attaches the servo on servoPin to the servo object
+  Serial.begin(9600);
+}
+
+void loop() {
+if (Serial.available() > 0) {
+    input = Serial.parseInt();        // reads the value from Serial
```

Comment:
With the line ending menu of Serial Monitor set to anything other than "No line ending", there is an issue. For example, if I enter 42 in the Serial Monitor input field, I get this output:
```
Servo set to 42 degree
Servo set to 0 degree
```
"Newline" is the default setting of this menu, so it's likely many users will encounter this problem.

The easiest solution would be to document the required line ending setting. Of course, many users won't read the directions, so they will still have the problem.

There is also the issue of the 1 s delay before the input takes effect. Not a huge issue, but it doesn't provide a very responsive feeling to the interface.

I explained the cause and described the fix for these issues here: https://github.com/arduino-libraries/Servo/pull/44#discussion_r395993389

Reply:
ok ill look into your suggestions

Reply:
@per1234 should I mention in the document to use no line ending or should I try improvising code to work for  newline?

Reply:
The issue still occurs with the line ending menu set to "Both NL & CR".

The reason is because serial communication is relatively slow. `Serial.parseInt()` returns as soon as it encounters a non-digit. So when `Serial.parseInt()` returns the NL has already been received, but the CR has not. The "remove the line endings" code you added removes the NL, then `Serial.available()` returns 0 so the `while` loop exits. On the next time through the `loop` function the CR is in the receive buffer, so `Serial.parseInt()` returns 0 (because that's what it does when it doesn't encounter any digits).

The solution is to add a short delay before the "remove the line endings" code to allow the CR to be received, as I demonstrated in https://github.com/arduino-libraries/Servo/pull/44#discussion_r395993389.